parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -789,6 +789,14 @@ retagContainerImage() {
     fi
 }
 
+labelContainerImage() {
+    CONTAINER_IMAGE_URL=$1
+    LABEL_KEY=$2
+    LABEL_VALUE=$3
+    echo "labeling image ${CONTAINER_IMAGE_URL} with ${LABEL_KEY}=${LABEL_VALUE} using ctr"
+    ctr --namespace k8s.io image label $CONTAINER_IMAGE_URL $LABEL_KEY=$LABEL_VALUE
```

Comment:
I know at this point we pretty much only use `ctr`, though other functions (like pullContainerImage) support multiple CLI's, should we consider doing that here as well?

Reply:
can we also add a set of shellspec tests?

Reply:
added comment to explain the limitation